data/api.py

```diff
@@ -153,12 +153,12 @@ def authorize(self, request, pk=None):
         serializer = JobTokensSerializer(job.run_token)
         return Response(serializer.data, status=status.HTTP_200_OK)
 
-    @detail_route(methods=['post'], serializer_class=JobSummarySerializer)
-    def summary(self, request, pk=None):
+    @detail_route(methods=['post'], serializer_class=JobUsageSerializer)
+    def live_usage(self, request, pk=None):
```

Comment:
Can we add a `url_path` argument to make this `live-usage`, so that it's consistent with the other URLs?